docs/CONTRIBUTING.md

```diff
@@ -1,113 +1,972 @@
+
+### 📝 Documentation and Review
+
+- [📖 Documentation Standards](#-documentation)
+- [🔄 Pull Request Process](#-pull-request-process)
+- [Code Review Guidelines](#code-review)
+
+### 🔬 Advanced Topics
+
+- [⚗️ Experimental Features](#️-experimental-changes)
+- [🚀 Release Process](#-release-process)
+- [🆘 Getting Support](#-getting-help)
+
+---
+
+## Prerequisites
+
+### Repository Access
+
+> [!NOTE] Leboncoin employees only
+> Please ensure you have push rights to this repository, rather than forking the repository for contributions. Follow the "Engineering Contribution" guide in the (To be defined, Android Guild or Foundation) space in Backstage to get access.
+
+This repository being public, you can directly contribute to it. However, we encourage you to fork the repository for your contributions to avoid conflicts when pulling as only contributors can push to the main repository.
+
+### Knowledge Requirements
+
+Before contributing to Spark, ensure you have:
+
+1. **Basic Jetpack Compose knowledge** - Understanding of `@Composable` functions, state management, and UI composition
+2. **Kotlin proficiency** - Comfortable with Kotlin syntax, data classes, and sealed classes
+3. **Android development experience** - Familiar with Android Studio, Gradle, and the Android build system
+4. **Version control skills** - Git workflow knowledge including branching, merging, and pull requests
+
+### Pre-contribution Checklist
+
+Before starting development:
+
+1. **Review architecture patterns** - Study the [Creating Components](#creating-components) section
+2. **Examine existing components** - Look at implementations in `/spark/src/main/kotlin/com/adevinta/spark/components/`
+3. **Verify issue tracking** - Ensure there's a corresponding GitHub issue for your planned changes
+4. **Understand testing requirements** - Review the [Testing Requirements](#testing) section
+
+---
+
+## Environment Setup
+
+Configure your development environment for Spark Android development. This section covers all required tools and dependencies.
+
+### Prerequisites
+
+1. **Install Android Studio**
+   - Download from [developer.android.com](https://developer.android.com/studio)
+   - Install with recommended settings
+
+2. **Configure Android SDK Command-line Tools**
+   - Go to Tools > SDK Manager
+   - In the "SDK Tools" tab, check "Android SDK Command-line Tools (latest)"
+   - Click "Apply" to install the selected components
+
+3. **Set up environment variables**
+
+   Run the appropriate commands for your shell:
+
+   ```bash
+   # For bash users
+   echo "export ANDROID_HOME=\"$HOME/Library/Android/sdk\"" >> ~/.bash_profile
+   echo "export ANDROID_SDK_ROOT=\"$HOME/Library/Android/sdk\"" >> ~/.bash_profile
+   source ~/.bash_profile
+
+   # For zsh users
+   echo "export ANDROID_HOME=\"$HOME/Library/Android/sdk\"" >> ~/.zshrc
+   echo "export ANDROID_SDK_ROOT=\"$HOME/Library/Android/sdk\"" >> ~/.zshrc
+   source ~/.zshrc
+   ```
```

Comment:
I don't think these are necessary. Android Studio already takes care of linking sdk tools.

Reply:
What since when we don't have to install them 😲?
I had to do it again 3 years ago when I changed my laptop 😭

Reply:
Android Studio prompts you to install them with a nice wizard during setup.
And usually, the SDK path is automatically configured in `local.properties` (`sdk.dir`).